distribution/docs/src/main/resources/content/_configuring/landing-page.adoc

```diff
@@ -1,9 +1,9 @@
 :title: Landing Page
 :type: configuration
 :status: published
-:parent: Miscellaneous Configurations
+:parent: Admin Theming Configurations
```

Comment:
If we change the title above, this will have to change as well

Reply:
fixed